### PR TITLE
Fix code names showing up in new translations, add docs

### PR DESCRIPTION
**Pull request by krassowski**
## References

Adds a minimum docs for `TranslationBundle`; ideally we would have that in developer documentation as mentioned in  https://github.com/jupyterlab/jupyterlab/pull/10762#discussion_r681679761

## Code changes

Fix a mix-up of `trans.__` with `trans._p` by removing context and keeping it as `trans.__`.

## User-facing changes

- "Rename Error" will show up correctly
- `syncCollapseState` will have a proper title

## Backwards-incompatible changes

None

### CODE REVIEW

**Review by goanpeca — APPROVED** [review]

File: packages/translation/src/tokens.ts [hunk 1]
```diff
@@ -34,10 +34,48 @@ export class TranslatorConnector
   private _translationsUrl: string;
 }
 
+/**
+ * Bundle of gettext-based translation function.
```
Comment (krassowski) [inline packages/translation/src/tokens.ts:38]:
```suggestion
 * Bundle of gettext-based translation functions.
```

**Review by fcollonval — APPROVED** [review]
Thanks @krassowski

**Comment by fcollonval** [comment]
@meeseeksmachine please backport to 3.1.x

**Comment by krassowski** [comment]
@meeseeksdev please backport to 3.1.x

**Comment by krassowski** [comment]
The bot handle config is strange...

**Pull request merged**